splink/profile_data.py

```diff
@@ -190,7 +190,16 @@ def _add_100_percentile_to_df_percentiles(percentile_rows):
     return percentile_rows
 
 
-def profile_columns(linker, column_expressions, top_n=10, bottom_n=10):
+def profile_columns(linker, column_expressions=None, top_n=10, bottom_n=10):
+
+    if column_expressions is None:
+
+        column_expressions = [
+            col.input_name
```

Comment:
Night that result in duplicate column names if there are multiple input datasets

Since the input data frames must be conformant , i think you only need to look at the first Data frame

Reply:
Might it be simpler to just make the input list unique?

Reply:
Yep- you're correct on this. It will try and find the columns from df_l in df_r and crash because those columns don't exist.

Reply:
For context of the thread- the newest commit _does_ only look at the first dataframe, as Robin suggested. The use case I have outlined here wouldn't be possible within how profile_columns currently works. Currently, profile_columns will error if the user calls a column that is only in one of the dataframes.

eg. 
```
from splink.duckdb.linker import DuckDBLinker
import pandas as pd

df = pd.DataFrame(
    [
        {"unique_id": 1, "test": 1},
    ]
)

df_2 = pd.DataFrame(
    [
        {"unique_id": 1, "test_2": 1},
    ]
)

linker = DuckDBLinker([df,df_2])

#Both of these will produce the same error

linker.profile_columns(["test"])
linker.profile_columns()
```

Thus, I believe the current implementation of this PR functions as intended. More broad changes would be needed if we wished to change this behaviour, and I don't believe we do.